blurb/flit.ini

```diff
@@ -0,0 +1,13 @@
+[metadata]
+module = blurb
+author = Python Core Developers
+author-email = core-workflow@python.org
+home-page = https://github.com/python/core-workflow/tree/master/blurb
+requires-python = >=3
+description-file = README.rst
+classifiers = Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
```

Comment:
Sure, I just didn't bother to check blurb to see what version it was restricted to as that's what Larry specified on PyPI.